src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -251,6 +252,12 @@ private ImmutableArray<ISymbol> GetSymbolsFromToken(SyntaxToken token, SolutionS
                 token = accessedMember;
             }
 
+            if (GetBindableNodeForTokenIndicatingCollectionExpression(token, out var collectionExpression))
+            {
+                ISymbol? symbol = semanticModel.GetTypeInfo(collectionExpression, cancellationToken).ConvertedType;
+                return symbol is not null ? ImmutableArray.Create(symbol) : ImmutableArray<ISymbol>.Empty;
+            }
+
             return semanticModel.GetSemanticInfo(token, services, cancellationToken)
```

Comment:
i'm surprised GetSemanticInfo isn't working here.  I would think it would get the expression parent and bind that.  can you look into that?

Reply:
Done. It's even easier :-) 
I was worried about `IsBindableToken` returning true for a bracket token, but it looks like the meaning is not that it's bindable but rather that we should bind its parent.

Reply:
yaay.